nltk/corpus/reader/nombank.py

```diff
@@ -58,6 +58,11 @@ def __init__(
             corresponding to this corpus.  These parse trees are
             necessary to resolve the tree pointers used by nombank.
         """
+        
+	    # If framefiles is specified as a regexp, expand it.
+        if isinstance(framefiles, string_types):
```

Comment:
Hi @alvations, I am not sure if we need to expand `framefiles` here when it is a regexp since `framefiles` will be expanded again from the following `CorpusReader.__init__(self, root, framefiles, encoding)`. Correct me if I am wrong.